pages/common/git-repl.md

```diff
@@ -0,0 +1,21 @@
+# git repl
+
+> Git REPL (read-evaluate-print-loop) - an interactive Git shell.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-repl>.
+
+- Start an interactive Git shell:
+
+`git repl`
+
+- Run a Git command while in the Git interactive shell:
+
+`{{git_subcommand}} {{command_arguments}}`
+
+- Run a command (not through Git) while in the Git interactive shell:
+
+`!{{command}}`
+
+- Exit the interactive shell (or press CTRL + D):
```

Comment:
I think we should use the same wording for all three descriptions here. I prefer `interactive Git shell` as we used in the first example.

Reply:
Done